src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/integration/CrsReferralsIntegrationTest.kt

```diff
@@ -8,7 +8,7 @@ import java.time.Period
 class CrsReferralsIntegrationTest : IntegrationTestBase() {
   @Test
   @Sql("classpath:testdata/sql/seed-pathway-statuses-1.sql")
-  fun `Get All CRS Referrals happy path`() {
+  fun `Get All CRS Referrals happy path - with caching`() {
```

Comment:
I think you should create at least 1 separate test using CacheManager and asserting the value is returned from the cache, not the mock. For example here:

https://www.baeldung.com/spring-data-testing-cacheable

Reply:
Good point - I'll create a new ticket to enhance these tests.